release/ReleaseNotes

```diff
@@ -3,8 +3,29 @@
 
 
 --------------------------------------------------------------------------------------------------------------
-ganga/python/Ganga
+ganga/python/GPIDev
 --------------------------------------------------------------------------------------------------------------
-* ...
+* Fix to MassStorageFile. The local area is now successfully cleaned up after files have been copied
+  to the mass storage area. Much improvement to the code.
+
+  --------------------------------------------------------------------------------------------------------------
+ganga/python/test
+--------------------------------------------------------------------------------------------------------------
+* Added better testing for MassStorageFile.
```

Comment:
I'm not sure that we need to tell the user about testing changes. We should try to keep these as useful information for user as to how things have changed or important bugs that have been fixed.